hooks/uncrustify.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, args):
         self.check_installed()
         self.parse_args(args)
         self.file_flag = "-f"
+        self.show_diff = "-q" not in self.args
```

Comment:
I prefer to keep the existing convention of \*_flag. So name it `quiet_flag`, and set it to `"q"`.